src/universal/modules/menu/components/MenuItem/MenuItemWithShortcuts.js

```diff
@@ -27,23 +27,22 @@ const activeStyle = css({
   }
 });
 
-const labelStyle = css({
+const Label = styled('div')(({hasIcon, disabled}) => ({
   ...textOverflow,
   fontSize: ui.menuItemFontSize,
   lineHeight: ui.menuItemHeight,
-  padding: `0 ${ui.menuGutterHorizontal}`
-});
-
-const iconLabelStyle = css({
-  paddingLeft: 0
-});
+  padding: `0 ${ui.menuGutterHorizontal}`,
+  paddingLeft: hasIcon && 0,
+  color: disabled && 'grey'
```

Comment:
@ackernaut you may want to change the color here. to see it, try to remove yourself from an org where you're the only billing leader